doc/sphinx/source/shared/mxnet_tutorial.rst

```diff
@@ -0,0 +1,101 @@
+.. shared/mxnet_tutorial.rst:
+
+
+Working with subgraphs on MXNet
+################################
+
+One DL framework with advancing efforts on shared graph optimizations is Apache 
+MXNet\*, where Intel has contributed efforts showing how to work with our
+nGraph Compiler stack as an `experimental backend`_. Our approach provides 
+**more opportunities** to start working with different kinds of graph 
+optimizations **than would be available to the MXNet framework alone**, for 
+reasons outlined in our `features`_ documentation.
+
+.. TODO : Link to latest on mxnet when/if they do this instead of linking to PR;
+   keep in mind this tutorial will still work  regardless of the merge status of 
+   the experimental backend if you already use the ngraph-mxnet Github repo 
+
+The Intel nGraph Compiler provides an in-memory abstraction layer for converting 
```

Comment:
`in-memory` is probably not relevant here (I'd just cut it)

Reply:
Done